### ISSUE TITLE
Label - editable label shows close button

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#5352]
I think this is an unexpected change from https://github.com/patternfly/patternfly/pull/5209. 

Specifically this block was removed:

https://github.com/patternfly/patternfly/pull/5209/files#diff-2c450c181adeb511f6c0e5b9bdaab0cc05d7ec94487293b9cac0329ff2e38a7eL375-L378

```diff
-
-    .pf-c-button {
-      visibility: hidden; // use visibility hidden to retain positioning
-    }
```

`visibility` is set there on purpose (not along with `display: none`) to hide the button but keep its layout on the page.

@thatblindgeye can you confirm?

**Comment by thatblindgeye** [comment]
Is there a specific example showing the bug? I'm not sure if I'm noticing it.

Looking at the latest releases on Org, Core and React example for an editable Label look similar to a few releases back. For both it looks like the close button just doesn't get rendered when `pf-m-editable-active` is applied.

**Comment by mcoker** [comment]
@thatblindgeye ah sorry, it took me a minute to find the example, too! You can see it here - https://pf5.patternfly.org/components/label-group/#editable-labels-label-active-dynamic-label-group

> For both it looks like the close button just doesn't get rendered when pf-m-editable-active is applied.

Ah hah, yep you're right. I think we just need to remove the close button from that editable label example linked above. @mattnolting can you confirm? Or is there a reason we would want to leave the close button there, but hide it with `visibility: hidden`? I put a PR up with that change - https://github.com/patternfly/patternfly/pull/5354

**Comment by thatblindgeye** [comment]
Closed via #5354
